client/src/App.js

```diff
@@ -5,7 +5,7 @@ import "bootstrap/dist/css/bootstrap.css";
 
 import { Route, BrowserRouter as Router, Switch } from "react-router-dom";
 
-import AppointmentCreateForm from "./components/AppointmentCreateForm";
+import AppointmentDataForm from "./components/AppointmentDataForm";
```

Comment:
Data is a generic term that doesn't tell you what's on it. You might as well call it AppointmentForm.

Reply:
Due to the destructuring I made, now we are going to have two components that all suit for this name. Could you provide me some idea about how to name these two components differently?

Reply:
As we talked yesterday, I just simply changed the name `AppointmentDataForm` to `CreateOrUpdateAppointment`.